test/utils.data.test.js

```diff
@@ -117,57 +117,57 @@ const testResourceStream = async (t, resource) => {
   t.deepEqual(rowsAsObjects[1], {number: '3', string: 'four', boolean: 'false'})
 }
 
-test('Resource class with path', async t => {
+test('File class with path', async t => {
   // With path
   const path_ = 'test/fixtures/sample.csv'
-  const res = utils.Resource.load(path_)
+  const res = utils.File.load(path_)
   await testResource(t, res)
 })
 
-test('Resource class with descriptor', async t => {
+test('File class with descriptor', async t => {
   const descriptor = {path: 'test/fixtures/sample.csv'}
-  const obj2 = utils.Resource.load(descriptor)
+  const obj2 = utils.File.load(descriptor)
   await testResource(t, obj2)
 })
 
-test('Resource with path and basePath', t => {
-  const obj3 = utils.Resource.load('sample.csv', {basePath: 'test/fixtures'})
+test('File with path and basePath', t => {
+  const obj3 = utils.File.load('sample.csv', {basePath: 'test/fixtures'})
   testResource(t, obj3)
 })
 
-test('Resource with inline JS data', async t => {
+test('File with inline JS data', async t => {
   const data = {
     name: 'abc'
   }
-  const resource = utils.Resource.load({data})
+  const resource = utils.File.load({data})
   t.is(resource.size, 14)
   const stream = await resource.stream()
   const out = await toArray(stream)
   t.is(out.toString(), JSON.stringify(data))
 })
 
-test('Resource with inline text (CSV) data', async t => {
+test('File with inline text (CSV) data', async t => {
   const data = `number,string,boolean
 1,two,true
 3,four,false
 `
   // To make it testable with testResource we add the path but it is not needed
-  const resource = utils.Resource.load({
+  const resource = utils.File.load({
     path: 'test/fixtures/sample.csv',
     format: 'csv',
     data
   })
   await testResource(t, resource)
```

Comment:
`testResource` => `testFile` but don't forget to rename the test function as well